engine/src/render/deferred_shading/plugin.cpp

```diff
@@ -60,7 +60,8 @@ namespace
                                           4]; // intended to be a float array, but std140 layout aligns array elements
                                               // to vec4 size; number of vec4s = ceiling(MaxCascades / 4 components)
         int numCascades;
-        int padding2[3];
+        int useShadowNormalOffsetBias;
+        int padding2[2]; // padding to 16 bytes
```

Comment:
```suggestion
        bool useShadowNormalOffsetBias;
        char padding2[11]; // padding to 16 bytes
```
Maybe in here you could use a `bool` instead and have the padding array be of `char` type to allow the same padding but making the variable type better suited.

Reply:
In GLSL, [bools declared in uniform blocks](http://www.opengl.org/registry/doc/glspec45.core.pdf#page=157) are represented as unsigned integers (32-bit wide). So the correct type would actually be `unsigned int`. Although `int` should be fine